packages/styles/scss/theme/_foundation.scss

```diff
@@ -61,6 +63,36 @@
 
   --ilo-color-purple: rgba(150, 10, 85, 1);
 
+  --ilo-color-notification-icon: var(--ilo-color-white);
+  --ilo-color-notification-type-error: var(--ilo-color-red);
+  --ilo-color-notification-type-info: var(--ilo-color-blue);
+  --ilo-color-notification-type-success: var(--ilo-color-green);
+  --ilo-color-notification-type-warning: var(--ilo-color-yellow);
+  --ilo-grey-light: rgba(237, 240, 242, 1);
+  --ilo-grey-ui: rgba(184, 196, 204, 1);
```

Comment:
if it's colors follow the color convension, if it's only used in one component make it component specific

Reply:
This color is used in many components so it should be in the foundation :)

Reply:
Than we should name it accordingly, like other colors

Reply:
Cool will do @GGKapanadze